### ISSUE TITLE
Feature Request:  A Walkthrough on how to generate the eks.img and relevant keys

### ISSUE THREAD

**Issue by thirdeye-yusuf** [madisongh/keystore#1]
Hi,

Forgive my ignorance and noobiness, if you have any time, I would be forever grateful if you could write up a walkthrough or guide on how you generate the keys, magic thingy, the header, combine it all etc.

Many Thanks

**Comment by thirdeye-yusuf** [comment]
oh and guessing I should change 16bit values in `app/keystore/include/app/keystore/vectors.h` ?
Can they be any value (16 bit hex)?

**Comment by madisongh** [comment]
Yeah, the README files are a bit sparse, aren't they?  I'll try and spend some time either updating them or putting some info on the wiki here.

And yes, the FV and IV can be any 16 bytes (not bits), but probably best to get them from a good random number generator.
